package-lock.json

```diff
@@ -14,6 +14,7 @@
         "@turf/turf": "^5.1.6",
         "archiver": "^5.1.0",
         "connect-pg-simple": "^6.1.0",
+        "convert-svg-to-png": "^0.6.4",
```

Comment:
this is actually not used. should be removed.

Reply:
done